src/pages/RssArticles.vue

```diff
@@ -107,6 +107,8 @@ onUnmounted(() => {
               <v-checkbox v-model="rssStore.filters.unread" :label="$t('rssArticles.filters.unread')" hide-details />
               <v-spacer />
               <v-btn :disabled="rssStore.unreadArticles.length === 0" :text="$t('rssArticles.markAllAsRead')" color="primary" @click="rssStore.markAllAsRead()" />
+              &nbsp;
+              <v-btn :disabled="rssStore.feeds.length === 0" :text="$t('rssArticles.refreshAllFeeds')" color="primary" @click="rssStore.refreshAllFeeds()" />
```

Comment:
Don't use HTML character entities.
I would wrap both buttons in a flex div with a gap of 8px instead.
Moreover this makes the responsive design completely messed up, so you could add a flex-column with the `$vuetify.display.mobile` prop.

```suggestion
              <div :class="{'d-flex gap': true, 'flex-column': $vuetify.display.mobile}">
                <v-btn :disabled="rssStore.unreadArticles.length === 0" :text="$t('rssArticles.markAllAsRead')" color="primary" @click="rssStore.markAllAsRead()" />
                <v-btn :disabled="rssStore.feeds.length === 0" :text="$t('rssArticles.refreshAllFeeds')" color="primary" @click="rssStore.refreshAllFeeds()" />
              </div>
```

And add the class in the component:

```css
.gap {
  gap: 8px;
}
```